docs/_includes/tests/dialog-subpage.html

```diff
@@ -1,18 +1,8 @@
----
-ds: 4
-layout: test
-title: Dialog Full
----
-<svg hidden>
-    <symbol id="icon-chevron-light-left" viewBox="4 0 32 32">
-        <path d="M28.176 1.327c-0.468-0.468-1.249-0.468-1.717 0l-14.049 14.049c-0.234 0.234-0.39 0.546-0.39 0.859s0.156 0.624 0.39 0.859l13.581 13.581c0.234 0.234 0.546 0.39 0.859 0.39s0.624-0.156 0.859-0.39c0.468-0.468 0.468-1.249 0-1.717l-12.722-12.722 13.19-13.19c0.468-0.468 0.468-1.249 0-1.717z"></path>
-    </symbol>
-</svg>
 <div class="test">
     <div role="dialog" class="dialog" id="dialog" aria-labelledby="dialog-title">
         <div class="dialog__window dialog__window--full" role="document">
-            <button class="dialog__back" type="button">
-                <svg aria-label="Close Dialog">
+            <button class="dialog__back" type="button" aria-label="Back">
+                <svg class="icon icon--chevron-left">
```

Comment:
Does this need `aria-hidden="true"` now?

Reply:
Yes! Great catch.

Reply:
Shouldn't affect the tests, though, but good to have it be consistent anyway.

Reply:
Yes we should add it for consistency sake.